src/cloud/ops.rs

```diff
@@ -99,30 +105,31 @@ pub async fn find_cloud_instance_by_name(
     Ok(Some(instance))
 }
 
-async fn wait_instance_create(
+async fn wait_instance_available_after_opertion(
     mut instance: CloudInstance,
     client: &CloudClient,
     quiet: bool,
+    operation: &str,
 ) -> anyhow::Result<CloudInstance> {
-    if !quiet && instance.status == "creating" {
-        print::echo!("Waiting for EdgeDB Cloud instance creation...");
+    if !quiet && instance.status == operation {
+        print::echo!("Waiting for result of EdgeDB Cloud instance", operation, "...");
     }
     let url = format!("orgs/{}/instances/{}", instance.org_slug, instance.name);
-    let deadline = Instant::now() + INSTANCE_CREATION_WAIT_TIME;
+    let deadline = Instant::now() + INSTANCE_OPERATION_WAIT_TIME;
```

Comment:
Consider using [progressbar (spinner)](https://docs.rs/indicatif/latest/indicatif/struct.ProgressBar.html#method.new_spinner) because waiting for 5 minutes is not something that users would expect.